### ISSUE TITLE
Adding new dataset

### ISSUE THREAD

**Issue by GrgicevicLukaNTNU** [WenjieDu/PyPOTS#129]
### 1. Feature description

Hi,

The data is from coastal base stations that monitor vessel traffic. Irregularly sampled and with missing values. Automatic identification system data in this case contains 8 columns and approximately 10 000 samples per vessel. There are approximately 200 vessels per .csv file. I have a dozen of them for the area in Norway near Ålesund.
[ais_20201118.zip](https://github.com/WenjieDu/PyPOTS/files/11535731/ais_20201118.zip)

### 2. Motivation

This could be a fun playground for data imputation and prediction.

### 3. Your contribution

I believe I could help with coding, but this would be my first pull request. I'm trying to implement the BRITS model for data imputation, so at the moment, I'm preparing it for training.

**Comment by WenjieDu** [comment]
Hi there 👋,

Thank you so much for your attention to PyPOTS! If you find PyPOTS helpful to your work, please star⭐️ this repository. Your star is your recognition, which can help more people notice PyPOTS and grow PyPOTS community. It matters and is definitely a kind of contribution to the community.

I have received your message and will respond ASAP. Thank you for your patience! 😃

Best,
Wenjie

**Comment by WenjieDu** [comment]
Hi Luka, thank you very much for making this proposal! You're welcome to join the development channel with us on Slack https://pypots-org.slack.com/. We can discuss further to assign you this feature implementing task and assist you to finish it.

**Comment by GrgicevicLukaNTNU** [comment]
Hi Wenjie,

I would need a link to this workspace with my email address, a QR code for example…


Best regards,



Luka Grgičević



Ph.D. Candidate, Department of ICT and Natural Sciences



Faculty of Information Technology and Electrical Engineering



Norwegian University of Science and Technology

________________________________

**Comment by WenjieDu** [comment]
I'm so sorry. I should share an invitation link and here it is https://join.slack.com/t/pypots-org/shared_invite/zt-1gq6ufwsi-p0OZdW~e9UW_IA4_f1OfxA

**Comment by WenjieDu** [comment]
Hey, Luka, your Vessel AIS dataset is available in [TSDB v0.0.9](https://github.com/WenjieDu/TSDB/releases/tag/v0.0.9). Let me know when your code is ready for a PR to make your data instantly available in PyPOTS.

**Comment by GrgicevicLukaNTNU** [comment]
Maybe I changed the learning rate for 'adam', batch size, and cuda devices when we trained the model on the cluster. Two gpu-s would be perfect.

**Comment by WenjieDu** [comment]
> Hey, Luka, your Vessel AIS dataset is available in [TSDB v0.0.9](https://github.com/WenjieDu/TSDB/releases/tag/v0.0.9). Let me know when your code is ready for a PR to make your data instantly available in PyPOTS.

As I mentioned above, the vessel AIS dataset has been integrated into TSDB and is publicly available. Now that you don't plan to publish anything related to using this dataset for imputation in PyPOTS, I'm closing this issue because all tasks have been finished. Good job and congrats, Luka! 👍 You're welcome to reopen it if you have further related content for discussion.
